TestUtils/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  -->
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think `xmlns:android="http://schemas.android.com/apk/res/android"` is not necessary.